lib/hound/helpers/element.ex

```diff
@@ -201,6 +201,9 @@ defmodule Hound.Helpers.Element do
   You can also pass the selector as a tuple.
 
       element_displayed?({:name, "example"})
+      
+  Note: If you'd like to check presence of elements in the DOM use `element?/2`, 
+  `element_displayed?/1` will only consider elements that are always present in the DOM, either in visible or hidden state.
```

Comment:
`element_displayed?` will only check if the element is visible/actionable to the user. So if you have a div1 on top of div2, then `element_displayed?(div2)` will be false. The second line can be better worded to reflect this.